### DISCUSSION TITLE
代码块右上角复制按钮样式显示待优化

### DISCUSSION THREAD

**Discussion by zuihou** [vuepress-theme-hope/vuepress-theme-hope#3225] [General]
1. 内容过长，有遮挡
<img width="888" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/8500410/8af59318-866d-48a7-846d-692a098930e5">

2. 鼠标移到代码块
<img width="213" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/8500410/8bdda6f2-14f5-4d7f-9cce-66bbceb47760">

3. 鼠标移到复制按钮 
<img width="288" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/8500410/608ec4f0-d934-4b7b-b7de-94ec7eae6efd">

**Comment by Mister-Hope** [comment]
https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/4f2b6858073eb5f62e827d7b7bbe260f54c34d17
